pkg/keycloak/phaseHandler.go

```diff
@@ -53,13 +53,18 @@ func (ph *phaseHandler) Initialise(sso *v1alpha1.Keycloak) (*v1alpha1.Keycloak,
 }
 
 func (ph *phaseHandler) Accepted(sso *v1alpha1.Keycloak) (*v1alpha1.Keycloak, error) {
-
+	var err error
 	kc := sso.DeepCopy()
-	if kc.Spec.AdminCredentials != "" {
-		return nil, nil
+	adminPwd := kc.Spec.AdminCredentials
+
+	if adminPwd == "" {
```

Comment:
Seeing that this is done whether or not adminPwd is "", do we need this if statement?